apps/src/code-studio/components/progress/course_progress.jsx

```diff
@@ -24,7 +24,7 @@ const CourseProgress = React.createClass({
   propTypes: {
     professionalLearningCourse: React.PropTypes.bool,
     focusAreaPositions: React.PropTypes.arrayOf(React.PropTypes.number),
-    stages: React.PropTypes.arrayOf(stageShape),
+    stages: React.PropTypes.arrayOf(stageShape)
```

Comment:
Note that this comma was previously added in https://github.com/code-dot-org/code-dot-org/pull/9293/files#diff-a04b10d3f83ac4aa14e208f43bdabeb7R27.

I don't have any preference for trailing commas or not, but churning back and forth seems bad 😄.